extensions/warp-blink-wrtc/Cargo.toml

```diff
@@ -26,6 +26,8 @@ serde_cbor = "0.11.2"
 tokio = { version = "1", features = ["full"] }
 warp = { path = "../../warp" }
 webrtc = "0.6.0"
+# if this fails to build on mac, run this: `brew install autoconf automake libtool pkg-config`
```

Comment:
```suggestion
# if this fails to build on mac, run this: `brew install autoconf automake libtool pkg-config`
```

This is only required to build on mac or is there any dependency that is required so it would run for those who are running the application itself?

Reply:
those dependencies are needed for any platform. it seems difficult to build on windows